CardScan/Classes/API.swift

```diff
@@ -190,6 +178,14 @@ public struct Api {
         }.resume()
     }
     
+    static func urlWithQueryParameters(baseUrl: String, endpoint: String, parameters: [String: Any]) -> URL? {
+        var components = URLComponents(string: baseUrl + endpoint)
+        components?.queryItems = parameters.map { (key, value) in URLQueryItem(name: key, value: value as? String) }
```

Comment:
Sorry, I should have caught this on my first read of the PR.

If you get a parameter that isn't a string it will silently fail. I suggest making `parameters` of type `[String: String]` or try to handle cases and error out if you get one that you aren't expecting instead of setting the value to nil